bls/Cargo.toml

```diff
@@ -6,24 +6,11 @@ authors = ["Grandine <info@grandine.io>"]
 [lints]
 workspace = true
 
-[dependencies]
-blst = { workspace = true }
-derivative = { workspace = true }
-derive_more = { workspace = true }
-fixed-hash = { workspace = true }
-hex = { workspace = true }
-impl-serde = { workspace = true }
-itertools = { workspace = true }
-once_cell = { workspace = true }
-rand = { workspace = true }
-serde = { workspace = true }
-serde_utils = { workspace = true }
-ssz = { workspace = true }
-static_assertions = { workspace = true }
-thiserror = { workspace = true }
-typenum = { workspace = true }
-zeroize = { workspace = true }
+[features]
+bls-blst = ["dep:bls-blst"]
```

Comment:
Is there a particular reason why `bls` prefix is used for feature names? I prefer just `blst`, `zkcrypto`. As I mentioned previously we most likely will try to also hook `rust-kzg` backends using the same switches.